src/main/java/pl/tomaszdziurko/jvm_bloggers/view/admin/moderation/ModerationPageRequestHandler.java

```diff
@@ -19,7 +21,8 @@
     @SpringBean
     private BlogPostRepository blogPostRepository;
 
-    public ModerationPageRequestHandler() { }
+    public ModerationPageRequestHandler() {
+    }
 
     @Autowired
     public ModerationPageRequestHandler(BlogPostRepository blogPostRepository) {
```

Comment:
The handcrafted constructor can be replaced with Lombok's `@AllArgsCanstructor(onConstructor = @__(@Autowired))`